src/test/java/com/puppycrawl/tools/checkstyle/AbstractPathTestSupport.java

```diff
@@ -63,7 +61,7 @@ protected final String getResourcePath(String filename) {
     protected static String readFile(String filename) throws IOException {
         return new String(Files.readAllBytes(
                 Paths.get(filename)), StandardCharsets.UTF_8)
-                .replaceAll(CRLF_REGEX, LF_REGEX);
+                .replaceAll(CRLF_REGEX, "");
```

Comment:
Do you actually mean
```java
XxxxUtil.toLfLineEnd(new String(Files.readAllBytes(
	                Paths.get(filename)), StandardCharsets.UTF_8));
```
?
Would maybe the constant rename I suggested above already make it clear enough?

Reply:
> Do you actually mean

yes

> Would maybe the constant rename I suggested above already make it clear enough?

My subjective opinion. It will not be clear.
```
(new String(Files.readAllBytes(
	                Paths.get(filename)), StandardCharsets.UTF_8)
	                .replaceAll(CR_FOLLOWED_BY_LF_REGEX, ""))
```
vs 
```
XxxxUtil.toLfLineEnd(new String(Files.readAllBytes(
	                Paths.get(filename)), StandardCharsets.UTF_8));
```

@rnveach and @pbludov , please vote.

Reply:
@Vampire , looks like in this PR, only you and me.
Please update code to use util method, I already forgot a trick of your regexp.

Reply:
There is no big trick, really.
The regexp is simply a `\r` followed by a `\n` or a `\\r` followed by a `\\n`.
The `\n` and `\\n` are matched by positive lookahead, so are not part of the match, so if you replace, you only replace the `\r` or `\\r`.
But yeah, I'll add such a utility method.

Reply:
done